app/controllers/api/assignments_controller.rb

```diff
@@ -5,50 +5,56 @@ class Api::AssignmentsController < ApplicationController
 
     def create
         authorize Assignment
-
-        created_assignments = []
-        created_action_items = []
-        participant_ids = bulk_assignment_params.fetch(:participant_ids, [])
-        action_items = bulk_assignment_params.fetch(:assignments, [])
-
-        if action_items.empty? || participant_ids.empty?
-            render json: { error: 'Action items and Participants must be populated'}, status: :unprocessable_entity
+        participant_ids_string = single_assignment_params.fetch(:participant_ids)
+        participant_ids = participant_ids_string.split(/,/)
```

Comment:
is there a reason the participant_ids are in string form instead of being a list of ids like before?

Reply:
See our slack chat on issues with formdata and lists. Basically, formdata turns it into a string